rust/plasma/src/main.rs

```diff
@@ -6,15 +6,14 @@
 
 #![no_std]
 #![feature(start)]
-#![feature(default_alloc_error_handler)]
```

Comment:
This feature is now on by default.

Reply:
Thanks! FYI the example is from the the [MOS-hardware crate](https://github.com/mlund/mos-hardware/blob/main/examples/mega65-plasma.rs) which is likely more up-to-date.